packages/react-conformance/src/isConformant.ts

```diff
@@ -1,15 +1,32 @@
-import * as React from 'react';
-import { IsConformantOptions, ConformanceTest } from './types';
+import { TestObject, TestingOptions } from './types';
+import { withCustomConfig } from 'react-docgen-typescript';
+import * as path from 'path';
+import { defaultTests } from './defaultTests';
 
-export function isConformant(
-  Component: React.ComponentType,
-  options: IsConformantOptions,
-  extraTests?: ConformanceTest[],
-) {
-  // const { constructorName, exportedAtTopLevel = true, packageRoot } = options;
-  // if (extraTests) {
-  //   extraTests.forEach((test: ConformanceTest) => {
-  //     test();
-  //   });
-  // }
+export function isConformant(componentPath: string, options: TestingOptions, extraTests?: TestObject) {
+  const tsconfigPath = path.join(process.cwd(), 'tsconfig.json');
+  const parser = withCustomConfig(tsconfigPath, {});
+  const components = parser.parse(componentPath);
+
+  if (components.length === 1) {
+    const componentInfo = components[0];
+
+    for (const test in defaultTests) {
```

Comment:
Instead of `for..in` it's better to use `for..of` on `Object.keys()` (available everywhere) or `Object.values()` or `Object.entries()` (available in Node context and newer browsers).
```
for (const test of Object.values(defaultTests) {
  test(componentInfo, componentPath, options);
}
```